src/robot/parsing/suitestructure.py

```diff
@@ -149,7 +149,11 @@ def _is_in_included_suites(self, name, incl_suites):
         return incl_suites.match(self._split_prefix(name))
 
     def _split_prefix(self, name):
-        return name.split('__', 1)[-1]
+        result = name.split('__', 1)[-1]
+        if result:
+            return result
+        return name
+
```

Comment:
Unnecessary empty line.

Reply:
Working on correcting the atests. If I understand it correctly, then we have also acceptance tests for this after correction. But I have to correct 33 Tests after, this takes a little while...